woodwork/accessor_utils.py

```diff
@@ -126,4 +123,14 @@ def _get_valid_dtype(series, logical_type):
     else:
         valid_dtype = logical_type.pandas_dtype
 
-    return valid_dtype
+    return str(valid_dtype)
+
+
+def _get_valid_ltype_dtype(series, logical_type):
```

Comment:
Added this helper because it was a code pattern I was seeing a fair amount, and I think it helps us differentiate when we need to be dealing with the actual series dtype (which will be `'object'` for certain koalas logical types) and when we should be dealing with the user-facing dtype (which should be `'str'` for those same columns).

@thehomebrewnerd I recognize this kind of puts us in another place of storing dtypes in multiple ways, but the sense I got from the Koalas PRs is that we don't really want users to interface with the `'object'` dtype more than necessary. So any time we are showing something to the user, I've made the change to use the `'str'` dtype--this caused changes in a couple of places (how we store physical type in serialization, the Column Accessor's repr, the physical type we present in descrive).